easybuild/easyconfigs/r/RStudio-Server/RStudio-Server-1.3.1093-foss-2020a-Java-11-R-4.0.0.eb

```diff
@@ -30,19 +38,28 @@ dependencies = [
     ('Java', '11', '', True),
 ]
 
-osdependencies = [
```

Comment:
These are not used by Compute Canada and I suspect they are not really necessary in a user space usage scenario.

Reply:
The exectuables are still linked against `libpam.so` so I think that these should actually remain